### DISCUSSION TITLE
centOS7 docker启用后为什么找不到节点

### DISCUSSION THREAD

**Discussion by tiefengsun** [sunyuting83/v2rss-go#10] [Q&A]
我环境是CentOS7 docker 安装并拉起之后，防火墙开启了。5500端口。为什么访问的页面是空白的。我在win下部署就没有问题。

**Comment by sunyuting83** [comment]
去拉dev分支的docker试试。

**Reply by tiefengsun** [reply]
我是个小白，能讲讲拉分支Dev具体怎么操作吗？谢谢。
我之前用centos直接安装也不行。一样的白页面。

**Reply by sunyuting83** [reply]
https://github.com/sunyuting83/v2rss-go/tree/dev
